package.json

```diff
@@ -1,11 +1,16 @@
 {
   "name": "breakdancer",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "A breakpoint tracking utility",
-  "main": "index.js",
+  "main": "lib/index.js",
```

Comment:
agree with this, but will also need a `browser` property that points to the `index.js` file